data/monster/monsters/dracola.xml

```diff
@@ -43,9 +43,6 @@
 		<attack name="lifedrain" interval="3000" chance="20" length="8" spread="3" min="-300" max="-600">
 			<attribute key="areaEffect" value="greenshimmer" />
 		</attack>
-		<attack name="drowncondition" interval="1000" chance="20" length="8" spread="3">
-			<attribute key="areaEffect" value="poff" />
-		</attack>
```

Comment:
cip removed this?

Reply:
I couldn't find any information about this attack for this monster and it wasn't defined with any damage so the spell had no effect. If you know this monster causes drown condition and the amount of total damage I can add it back.